auth/auth.go

```diff
@@ -92,7 +94,8 @@ func Authenticate(login, password, ldap string) map[string]interface{} {
 	// attempt the authentication
 	if err != nil || !ok {
 		log.Println(err)
-		return u.BuildResponse("Invalid login credentials. Please try again")
+		u.SendForbidden(writer, map[string]interface{}{"error": err.Error()})
```

Comment:
Or should we rather send 401??

Reply:
Please, don't use interface directly. There is function that already make consistent response object.

Reply:
Yeah, 401 will be better. Because `The request has not been applied because it lacks valid authentication credentials for the target resource.`

Reply:
It seemed to me more straightforward, but OK. 

Ad 401 - OK.
Thanks